api/v1/openapi.yaml

```diff
@@ -1751,6 +1751,11 @@ definitions:
                 - SNAT
                 - DSR
                 - HYBRID
+              algorithm:
+                type: string
+                enum:
+                - RANDOM
+                - MAGLEV
```

Comment:
:-) I guess we don't. I added this in capital letters to be consistent with the other output there. Are we allowed to change the others, too or is this considered breakage? Presumably it might be okay to change these values, so we could fix up the others consistently?

Reply:
kk, no yelling :smile:

Reply:
Added here to discuss wrt breakage https://github.com/cilium/cilium/issues/13138

Reply:
Actually we can't. go-swagger converts enums to capital case. So, to compare a value with enum we need to call `strings.ToLower(..)` which is error prone.